src/components/ae-fraction/ae-fraction.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <span class="ae-fraction" :class="[ fill ]">
+    <sup>{{ numerator }}</sup>
+    &frasl;
+    <sub>{{ denominator }}</sub>
+  </span>
+</template>
+<script>
+export default {
+  name: 'ae-fraction',
+  props: {
+    /**
+     * Fill property changes the color state of the fraction component.
+     * Select something between: `primary, secondary, neutral, alternative, white, black`
+     */
+    fill: {
+      type: String,
+      validator: value => [
+        'primary',
+        'secondary',
+        'neutral',
+        'alternative',
+        'white',
+        'black',
+      ].includes(value),
+    },
+
+    /**
+     * The numerator property defines
+     * the upper number in the fraction.
+     */
+    numerator: {
+      type: [Number, String],
+      default: 0,
+    },
+
+    /**
+     * The numerator property defines
+     * the lower number in the fraction.
+     */
+    denominator: {
+      type: [Number, String],
+      default: 0,
+    },
+  },
+};
+</script>
+<style lang="scss" scoped>
+@import '../../styles/variables/colors';
+@import '../../styles/variables/typography';
+
+.ae-fraction {
+  display: inline-block;
```

Comment:
`sup` and `sub` are inline elements, isn't it? Looks like This component hasn't any `block` content.

Reply:
yeah, so?

Reply:
let's remove this line

Reply:
I would like the behaviour of this element to be inline-block, and not simply inline.

Reply:
But I wouldn't like, what is the reasons for inline-block in this case?

Reply:
Its not a matter of like or not! The way I want this component to behave is as an `inline-block` and not an inline element. I would like it to have both horizontal and vertical spacing.

Reply:
Please provide an example how it can be used.